onload/build/deviceplugin/Dockerfile

```diff
@@ -2,8 +2,12 @@
 # SPDX-FileCopyrightText: (c) Copyright 2023 Advanced Micro Devices, Inc.
 FROM golang:1.20.4 AS builder
 
+ARG KUBERNETES_ONLOAD_LOCATION
+
 WORKDIR /app
-RUN git clone --depth 1 https://github.com/Xilinx-CNS/kubernetes-onload.git
+ADD ${KUBERNETES_ONLOAD_LOCATION} kubernetes-onload.tar.gz
+RUN mkdir -p /app/kubernetes-onload
+RUN tar xzf kubernetes-onload.tar.gz -C /app/kubernetes-onload --strip-components=1
```

Comment:
I don't know if lines 9 & 10 are modeled on a container definition I'd done earlier but that is indeed why I had done it like that elsewhere - it was so that it doesn't matter what the archive is called and what the top level directory name is; these vary wildly with automatic archive generation and it then means you can source a wider variety of tarballs. Otherwise doing it all with ADD would be quite neat.